session/src/main/java/org/apache/iotdb/session/pool/SessionPool.java

```diff
@@ -625,41 +625,7 @@ public void insertOneDeviceRecords(
       Session session = getSession();
       try {
         session.insertRecordsOfOneDevice(
```

Comment:
two overload function which only differs in `haveSorted param`, thus merged to one